src/slides/redux-intro/action-example.js

```diff
@@ -0,0 +1,6 @@
+const ADD_TODO = { type: "ADD_TODO", text: "Go to swimming pool" };
```

Comment:
Are we starting with these actions to get the point across and moving towards flux standard actions later on?

Reply:
Yeah exactly, the idea is to show that actions are plain JS objects that _have_ to have a `type` property but nothing else is mandatory. I was thinking we could add suggested action object shapes in a more in depth section on actions.

Reply:
Perfect. In that case :shipit: